source/fundamentals/crud/read-operations/geo.txt

```diff
@@ -0,0 +1,306 @@
+===================
+Search Geospatially
+===================
+
+.. default-domain:: mongodb
+
+.. contents:: On this page
+   :local:
+   :backlinks: none
+   :depth: 2
+   :class: singlecol
+
+Overview
+--------
+
+On this page, you can learn how to query **geospatial data**
+with the MongoDB Java Driver. 
+
+Geospatial data is data tied to a geographical location on
+the surface of the Earth. Examples of geospatial data include:
+
+- Locations of movie theaters
+- Borders of countries
+- Routes of bicycle rides
+- Dog exercise areas in New York City
+
+You should read this guide if you would like to:
+
+- Understand the different geospatial data formats in MongoDB.
+- Build geospatial queries.
+
+Coordinates on an Earth-like Sphere
+-----------------------------------
+
+You can represent geospatial data on an Earth-like sphere with the **GeoJSON**
+format. GeoJSON is a geospatial data format. You can use the GeoJSON format to
+store your data, and you must use the GeoJSON format to query your geospatial
+data with MongoDB. 
+
+
+For a concrete reference, GeoJSON data often resembles the following structure:
+
+.. code-block:: json
+
+   "<field name>" : {
+      "type": "<GeoJSON type>",
+      "coordinates": [
+         [<longitude>, <latitude>],
+         ...
+         [<longitude>, <latitude>]
+      ]
+   }
+
+For definitive information on GeoJSON, see the
+`official IETF GeoJSON specification <https://tools.ietf.org/html/rfc7946>`__.
+
+.. external resource
+
+
+GeoJSON Types
+~~~~~~~~~~~~~
+
+Your GeoJSON object's type determines its geometric shape.
```

Comment:
Following with the available types might make it flow more naturally, and help define the shapes.

Reply:
Thanks for the note, after rereading this a few times I agree that going Type -> Position -> Type is awkward and hard to understand. I have moved "GeoJSON" position to its own section. Fix: 

Going forward, I will try to better get a sense for when a concept is progressing linearly vs. switching back and forth between something else.